packages/babel-plugin-syntax-bigint/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "babel-plugin-syntax-bigint",
```

Comment:
The `plugin-syntax-bigint` is already on master, could we remove it from this PR?